packs/datadog_agent/templates/pack.nomad.tpl

```diff
@@ -0,0 +1,146 @@
+[[- $ports := .my.ports ]]
+
+# see https://developer.hashicorp.com/nomad/docs/job-specification/job
+job "[[ .my.job_name ]]" {
+  region      = "[[ .my.region ]]"
+  datacenters = [[ .my.datacenters | toJson ]]
+  type        = "service"
```

Comment:
shouldn't it be batch, to run on all Nomad clients?

Reply:
Preferably, no.

The Agent is supposed to be long-lived:

> The service scheduler is designed for scheduling long lived services that should never go down.

Reply:
Fixed this!